msal/src/main/java/com/microsoft/identity/client/AccountAdapter.java

```diff
@@ -98,8 +118,13 @@ public List<ICacheRecord> filter(@NonNull final List<ICacheRecord> records) {
 
         private boolean hasNoCorrespondingHomeAccount(@NonNull final ICacheRecord guestRecord,
                                                       @NonNull final List<ICacheRecord> homeRecords) {
+            final String methodTag = CacheRecordFilter.class.getSimpleName() + ":hasNoCorrespondingHomeAccount";
```

Comment:
We usually define a class level TAG constant and then append that to method name to create a method tag (rather than doing it inline here)

Reply:
addressed... added TAG to each inner class of this AccountAdapter class